app/user/serializers.py

```diff
@@ -110,7 +95,7 @@ def validate(self, data: Any) -> Any:
 
     def save(self, **kwargs: Any) -> Any:
         user_id = force_str(
-            urlsafe_base64_decode(self.validated_data.get("uidb64"))
+            urlsafe_base64_decode(self.validated_data.get("encoded_pk"))
         )
         user = User.objects.get(pk=user_id)
         user.is_verified = True
```

Comment:
where are you confirming that the token is valid?

Reply:
To ensure the token is sent to the right user